src/boot/efi/pe.c

```diff
@@ -140,48 +140,49 @@ static size_t section_table_offset(const DosFileHeader *dos, const PeFileHeader
         return dos->ExeHeader + offsetof(PeFileHeader, OptionalHeader) + pe->FileHeader.SizeOfOptionalHeader;
 }
 
-static void locate_sections(
+static ssize_t locate_section(
                 const PeSectionHeader section_table[],
                 size_t n_table,
-                const char * const sections[],
-                size_t *offsets,
-                size_t *sizes,
+                const char *section_name,
+                size_t *offset,
+                size_t *size,
```

Comment:
as per coding style we name return parameters `ret_xyz`. if you touch code that didn't follow this style, please update.

https://systemd.io/CODING_STYLE